src/lib.rs

```diff
@@ -208,6 +222,18 @@ impl<'a> Ammonia<'a> {
             child.parent.replace(None);
             dom.append(&parent.clone(), NodeOrText::AppendNode(child.clone()));
         }
+        pass
+    }
+
+    fn fix_child(&self, child: &mut Handle, link_rel: &Option<StrTendril>) {
```

Comment:
I am passing the whole config struct (that's self).

The reason there's also a rel parameter is that the rel gets converted from a str to a tendril.